src/main/java/uk/gov/hmcts/reform/prl/services/citizen/CitizenCaseUpdateService.java

```diff
@@ -186,6 +192,26 @@ public CaseDetails submitCitizenC100Application(String authToken,
         return partyLevelCaseFlagsService.generateAndStoreCaseFlags(String.valueOf(caseDetails.getId()));
     }
 
+    public void updatePcqIdForMainApplicant(CaseData citizenUpdatedCaseData, CaseData caseDataToSubmit) {
+        if (StringUtils.isNotEmpty(citizenUpdatedCaseData.getC100RebuildData().getApplicantPcqId())) {
+            if (C100_CASE_TYPE.equalsIgnoreCase(CaseUtils.getCaseTypeOfApplication(caseDataToSubmit))) {
+                checkAndupdatePcqIdForParty(caseDataToSubmit.getApplicants().get(0).getValue(), citizenUpdatedCaseData);
+            } else {
+                checkAndupdatePcqIdForParty(caseDataToSubmit.getApplicantsFL401(), citizenUpdatedCaseData);
+            }
```

Comment:
We dont need it for DA. Also this case data is never supposed to have case type of application as anything apart from C100